docs/python/testing.md

```diff
@@ -163,6 +163,25 @@ If discovery fails (for example, the test framework isn't installed or you have
 
 ![Discovery failure error messaged displayed in the Test Explorer](images/testing/test-discovery-error.png)
 
+### Django unittests
+
+The Python extension also offers support for running Django unittests! However, if you are writing tests for your Django application, there are a few additional steps needed in order to have your tests discovered.
+
+First, ensure you have correctly set `"python.testing.unittestEnabled": true,` in your `settings.json` file, then follow the steps below to add `MANAGE_PY_PATH` as an environment variable:
+
+- Create a file named `.env` at the root of your project.
+- Add `MANAGE_PY_PATH='path-string-to-manange.py-path'` to the `.env` file, replacing `path-string-to-manange.py-path` with the path to your application's `manage.py` file (Note: you can copy the path by right clicking on the file and selecting **Copy Path**).
+- Add the following to your `settings.json` file `"python.envFile": "${workspaceFolder}/.env”` to incorporate this file in your environment (edit path as necessary if `.env` is not at root).
+- Add Django args to `"python.testing.unittestArgs": []` in `settings.json` as needed, and remove any arguments that are not compatible with Django.
+
+Navigate to the Testing Panel, and refresh the tree-view to see your Django tests populate!
+
+In the case your tests are not discovered, try the following troubleshooting steps:
```

Comment:
I'm wondering if it'd be worth adding a Troubleshooting section like we do for [Linting ](https://code.visualstudio.com/docs/python/linting#_troubleshooting-linting) and [Formatting](https://code.visualstudio.com/docs/python/formatting#_troubleshoot-formatting)?

Reply:
Hmmm I am not opposed to it, however I am not sure if it would fit in the same table/labels as they have there since the issue would all be the tests aren't getting discovered and there are just a list of things to try. Are you thinking just having one row where the issue is "Tests are not getting discovered" and then list the possible solutions together?

Reply:
We could try framing it like this (with some cleaning up lol). 
| Problem | solution |
|----|----|
| There are configuration errors | Look at the Python Output channel for any error messages that may have triggered | 
| Commands that are being run are not the ones intended | Try running commands in the Command Line, then transfer the same command to how it is represented in VS Code. For example, you may run `python manage.py test --arg` in the Command Line which is represented by setting `MANAGE_PY_PATH='./manage.py'` and `"python.testing.unittestArgs": [--arg]` in VS Code. Check the output as it should print the commands that are actually ran. |
| The path to manage.py may be incorrect. | Attempt using the absolute path for `manage.py`, if you initially tried using the relative path. |

Reply:
Yeah I think it is hard to say if these problems will appear differently for users- I think they will just see nothing show up or and error and multiple steps could be the solution for that